ci/jenkins/Jenkinsfile_centos_cpu

```diff
@@ -21,7 +21,7 @@
 // See documents at https://jenkins.io/doc/book/pipeline/jenkinsfile/
 
 // timeout in minutes
-max_time = 180
+max_time = 240
```

Comment:
why increase the timeout? http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Funix-gpu/detail/PR-18025/59/pipeline/ 

Now the jobs hang even longer before termination

Reply:
I noticed that windows tests were running into timeout. also, we should have other ways for handling hanging issue than relying on step level timeout.